DirectProgramming/DPC++FPGA/ReferenceDesigns/qrd/README.md

```diff
@@ -21,7 +21,7 @@ Please refer to the performance disclaimer at the end of this README.
 
 | Device                                         | Throughput
 |:---                                            |:---
-| Intel® PAC with Intel Arria® 10 GX FPGA        | 24k matrices/s for matrices of size 128 * 128
+| Intel® PAC with Intel Arria® 10 GX FPGA        | 24.7k matrices/s for matrices of size 128 * 128
```

Comment:
I don't think we should update this number.  24k is what we are committing to as a minimum, I don't think we want to change that because we find a slightly better seed.

Reply:
The required throughput is apparently 24.5k so I'm setting it correctly now.